packages/factory_reset_tools/lib/dbus/reset_media.dart

```diff
@@ -10,7 +11,22 @@ enum ResetMediaCreationStatus {
   copying,
   finalizing,
   finished,
-  failed
+  failed;
+
+  String displayName(FactoryResetToolsLocalizations lang) {
```

Comment:
nitpick: would be slightly more readable with a switch expression

Reply:
True! I always forget about new features when I'm coding on autopilot 😄